src/snowflake/snowpark/_internal/analyzer/schema_utils.py

```diff
@@ -16,43 +16,43 @@
 
 
 def command_attributes() -> List[Attribute]:
-    return [Attribute('"status"', StringType())]
+    return [Attribute('"status"', StringType(StringType._MAX_LENGTH))]
```

Comment:
On server, these columns are not set with `STRING(LENGTH)` so the default value it takes is `STRING(16777216)`. On the client we have to explicitly set it to `StringType(_MAX_LENGTH)` because here `StringType() != StringType(_MAX_LENGTH)`.